common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/MicrosoftTokenResponse.java

```diff
@@ -91,8 +91,8 @@ public String getSessionKeyJwe() {
      * @param sesionKey the session key JWE associated with this result, or null if none.
      * @return
      */
-    public String setSessionKeyJwe(String sesionKey) {
-        return mSessionKeyJwe;
```

Comment:
The old code here looks very strange 😁

Reply:
yes, seems was written this way by mistake. It was never used before so remained uncaught.